pkg/kubelet/dockertools/manager.go

```diff
@@ -1595,17 +1618,18 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubetypes.Docker
 
 	// No pod secrets for the infra container.
 	// The message isnt needed for the Infra container
-	if err, _ := dm.imagePuller.PullImage(pod, container, nil); err != nil {
-		return "", err
+	// NOTE: (random-liu) Should we just return this for compliance
```

Comment:
As the comment above said that the *message* is not needed for the Infra container.
However, SyncPod() needs *error* and *message*, and some other functions like `PullImage`, `doBackOff` all return *error* and *message* now, so we should return *message* here to be compliance with the other functions.